src/pages/Check.jsx

```diff
@@ -1,7 +1,14 @@
-const Check = () => (
-  <div>
-    <h1>Check</h1>
-  </div>
-)
+import { useState } from 'react'
+import Modal from '../components/Modal'
+
+const Check = () => {
+  const [modState, changeModState] = useState(true)
+  return (
+    <div>
+      <h1>Check</h1>
+      <Modal state={modState} statechange={changeModState}></Modal>
```

Comment:
These prop names are not descriptive, please use another set of prop names

Reply:
also with this two